### PR TITLE
Add go13 formula

### PR DESCRIPTION
**Pull request by rasky**

### CODE REVIEW

File: go13.rb [hunk 1]
```diff
@@ -0,0 +1,96 @@
+require 'formula'
+
+class Go13 < Formula
+  homepage 'http://golang.org'
+  head 'https://go.googlecode.com/hg/'
```
Comment (jacknagel) [inline go13.rb]: remove head

File: go13.rb [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+        ['windows', ['386', 'amd64']],
+        ['darwin',  ['386', 'amd64']],
+      ]
+    else
+      targets = [['darwin', ['']]]
+    end
+
+    # The version check is due to:
+    # http://codereview.appspot.com/5654068
+    (buildpath/'VERSION').write('default') if build.head?
```
Comment (jacknagel) [inline go13.rb]: remove head-related things

**Comment by rasky** [comment]
done

**Comment by DomT4** [comment]
Hi! If this is still useful please address any comments ` brew audit --strict ` has locally Also please deprecate the option names to match the current wording in Homebrew/Homebrew. Thanks!

**Comment by DomT4** [comment]
Closing per inactivity. Please feel free to request reopening or open a new PR if you're still interested - If you do, please fix any comments ` brew audit --strict ` has and rebase this against recent `godoc` and ` vet ` changes in Homebrew/homebrew, Thanks!

**Pull request closed without merging**